docs/how-to/service-protection-setup.md

```diff
@@ -0,0 +1,74 @@
+# Installing Kuadrant Service Protection into an existing OCM Managed Cluster
+
+## Introduction
+This walkthrough will show you how to install and setup the Kuadrant Operator into an [OCM](https://open-cluster-management.io/) [Managed Cluster](https://open-cluster-management.io/concepts/managedcluster/).
+
+## Prerequisites
+* Access to an Open Cluster Management (>= v0.11.0) Managed Cluster, which has already been bootstrapped and registered with a hub cluster
+  * We have [a guide](./mgc-setup.md) which covers this in detail
+  * Also see:
+    * https://open-cluster-management.io/getting-started/quick-start/
+    * https://open-cluster-management.io/concepts/managedcluster/
+* OLM will need to be installed into the ManagedCluster where you want to run the Kuadrant Service Protection components
+  * See https://olm.operatorframework.io/docs/getting-started/
+* Kuadrant uses Istio as a Gateway API provider - this will need to be installed into the data plane clusters
+  * See https://istio.io/v1.16/blog/2022/getting-started-gtwapi/
+  * We recommend installing Istio 1.17.0
+
+
+Alternatively, if you'd like to quickly get started locally, without having to worry to much about the pre-requisites, take a look at [this guide](./ocm-control-plane-walkthrough.md). It will get you setup with Kind, OLM, OCM & Kuadrant in a few short steps.
+
+
+## Install the Kuadrant OCM Add-On
+
+To install the Kuadrant Service Protection components into a `ManagedCluster`, target your cluster and run:
+
+```bash
+kubectl apply -f - <<EOF
+apiVersion: addon.open-cluster-management.io/v1alpha1
+kind: ManagedClusterAddOn
+metadata:
+ name: kuadrant-addon
+ namespace: kind-mgc-workload-1
+spec:
+ installNamespace: open-cluster-management-agent-addon
+EOF
+```
+
+**Note:** if you've run our Quickstart Setup guide, you'll  be set to run this command as-is.
```

Comment:
Should we include a link for the reference `Quickstart Setup` so it's clear what's being referred to here? What if they haven't run that? are there other steps needed?